Doc/library/urllib.request.rst

```diff
@@ -1249,7 +1249,10 @@ It is also possible to achieve the same result without using the
 
    >>> import urllib.request
    >>> f = urllib.request.urlopen('http://www.python.org/')
-   >>> print(f.read(100).decode('utf-8'))
+   >>> try:
```

Comment:
Why not `with`?

Reply:
It is the example without context manager.